manifests/0000_50_olm_15-packageserver.clusterserviceversion.yaml

```diff
@@ -9,6 +9,9 @@ metadata:
   annotations:
     include.release.openshift.io/self-managed-high-availability: "true"
 spec:
+  cleanup:
+    enabled: false
+  customresourcedefinitions: {}
```

Comment:
I think we'll also want to add this to the helm chart that's responsible for rendering the list of CVO manifests.

Here's an example of PR that makes those changes: https://github.com/operator-framework/operator-lifecycle-manager/commit/44ee08c8d0db139b1a36ed782d5ddcf01e427fdf.

There should be a Makefile command that renders the helm chart and dumps out the static YAML manifests.

Reply:
Done, however `make release` has updated quite a few other manifests - could you have a look if these are relevant and I should squash the commits?